File: Android/14 - Banco/app/src/main/java/com/example/ifpb/banco/Pessoa.kt
package com.example.ifpb.banco

/**
 * Created by ifpb on 17/05/18.
 */

class Pessoa (var nome: String, var idade: Int, var id: Int = -1){

    override fun toString(): String {
        return "${this.nome} - ${this.idade}"
    }
}

File: iOS/02 - Somador/Somador/Pessoa.swift
//
//  Pessoa.swift
//  Somador
//
//  Created by IFPB on 29/06/18.
//  Copyright © 2018 IFPB. All rights reserved.
//

import Foundation

class Pessoa: NSObject {
    var nome: String!
    
    override var description: String{
        return self.nome
    }
    
    init(nome: String) {
        self.nome = nome
    }
}
